lib/browser.js

```diff
@@ -112,8 +112,13 @@ var Browser = function (id, fullName, /* capturedBrowsers */ collection, emitter
       emitter.emit('browser_log', this, info.log, info.type)
     }
 
+    if (helper.isDefined(info.total)) {
+      this.lastResult.total = info.total;
```

Comment:
What happens if the adapter fails to call onStart()?  I guess we fail ugly here.  That's probably ok since it's on the adapter author not users.